searx/static/themes/simple/src/less/autocomplete.less

```diff
@@ -51,15 +51,32 @@
     display: block;
     background-color: var(--color-autocomplete-background);
     color: var(--color-autocomplete-font);
-    border: 1px solid var(--color-autocomplete-border);
     max-height: 32rem;
     overflow-y: auto;
     z-index: 100;
-    margin-top: 3.2rem;
+    margin-top: 3.1rem;
+    border-radius: 0 0 10px 10px;
 
     &:empty {
       display: none;
     }
+
+    & ~ #q {  // Doesn't work --> autocomplete must be above #q, #send_search and #clear_search
```

Comment:
we are using the `autocomplte.js` lib, which is quite old, this is AFAIK not possible, but maybe @dalf knows more...